lib/project.rb

```diff
@@ -71,8 +71,8 @@ def log_exception(exception, level)
     def init_multilog(verbose)
       @temp_pre_logger = StringIO.new
       @pre_logger = MonoLogger.new(@temp_pre_logger)
-      @stdout_logger = MonoLogger.new($stdout)
-      @logger = MultiLogger.new(@pre_logger, @stdout_logger)
+      @stderr_logger = MonoLogger.new($stderr)
+      @logger = MultiLogger.new(@pre_logger, @stderr_logger)
```

Comment:
why do we need to use stderr instead of stdout?

Reply:
In this case please update lib/config/sentry.rb:
```
 MonoLogger.new($stdout).warn('SENTRY_TRANSPORT_SSL_CA environment specified, but file does not exist')
 ```

Reply:
Done tree-wide search with commit a241bce6ce3b99926c4d5f96bcc5d20cef74e272; actually the error with `SENTRY_TRANSPORT_SSL_CA` was preventing running AutoHCK for Vitalii.